### PR TITLE
SWARM-1630: Dependency cleanup for MicroProfile

### PR DESCRIPTION
**Pull request by kenfinnigan**
- [x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [x] Have you built the project locally prior to submission with `mvn clean install`?

-----

Motivation
----------
Verify and cleanup dependencies for new MicroProfile fractions.

Modifications
-------------
Update module.conf for MP FT to remove unnecessary dependencies.

Result
------
No impact to product.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1000,12 +1000,12 @@
     <module>fractions/wildfly/security</module>
 
     <module>fractions/keycloak</module>
-    <module>fractions/microprofile/microprofile</module>
     <module>fractions/microprofile/microprofile-config</module>
     <module>fractions/microprofile/microprofile-health</module>
     <module>fractions/microprofile/microprofile-jwt</module>
     <module>fractions/microprofile/microprofile-metrics</module>
     <module>fractions/microprofile/microprofile-fault-tolerance</module>
+    <module>fractions/microprofile/microprofile</module>
```
Comment (mkouba) [inline pom.xml:1008]: What's the reason for this change?

Comment (kenfinnigan) [reply pom.xml:1008]: Maven builds it in that order anyway, so figured it made sense to arrange them such as /microprofile has the others as dependencies

**Review by mkouba — APPROVED** [review]
